packages/bot-strategy-runner/CHANGELOG.md

```diff
@@ -1,242 +0,0 @@
-# Change Log
```

Comment:
nit: what are all of these deletions? Are these related?

Reply:
> nit: what are all of these deletions? Are these related?

review this first: https://github.com/UMAprotocol/protocol/pull/4309

that's why this PR is in draft